jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -370,12 +370,19 @@ def plot_spectral_line(self, line, plot_units=None, **kwargs):
         if plot_units is None:
             plot_units = self.data()[0].spectral_axis.unit
 
+        if isinstance(line["colors"], str):
+            line_color = [line["colors"]]
+        # TODO: Figure out why this is sometimes a multi-element table Column
+        # instead of a single value so we can remove (or improve) this workaround.
+        elif isinstance(line["colors"], table.Column):
+            line_color = [line["colors"][0]]
```

Comment:
Can you trigger this consistently? If so, change this line to throw exception and follow the traceback.

Reply:
I haven't managed to trigger it at all, this workaround is from following @camipacifici's traceback.

Reply:
And the traceback just leads back to the `plot_spectral_line` call, it doesn't show how the value is getting retrieved as a column instead of a single value.